integ-test/src/test/java/com/amazon/opendistroforelasticsearch/sql/ppl/SearchCommandIT.java

```diff
@@ -0,0 +1,60 @@
+/*
+ *   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.sql.ppl;
+
+import java.io.IOException;
+import org.elasticsearch.client.ResponseException;
+import org.junit.Test;
+import static com.amazon.opendistroforelasticsearch.sql.esintgtest.TestsConstants.TEST_INDEX_BANK;
+
+public class SearchCommandIT extends PPLIntegTestCase {
+
+    @Override
+    public void init() throws IOException {
+        loadIndex(Index.BANK);
+    }
+
+    @Test
+    public void testSearchAllFields() throws IOException {
+        String result = executeQuery(String.format("search source=%s", TEST_INDEX_BANK));
+        assertTrue(result.contains("\"total\": 7"));
```

Comment:
Is it possible to assert all the response instead of just one field. e.g.
schema(contains("name", "string")))
or we can assert the full string match.
Actually, we don't need much data for integration test purpose, IMO, the sample data include all the types, null, missing should be enough for IT purpose.

Reply:
I'm considering to add a utils to build result set and schema from the result, so that we can test the behavior in a more accurate way.
For the sample data, what about applying `bank_with_null_values` (already in our resources) to all tests?